### DISCUSSION TITLE
“夸克（Quark）”迷你开发者套件 | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1398] [Q&A]
# “夸克（Quark）”迷你开发者套件 | Seeed Studio Wiki

“夸克（Quark）”迷你开发者套件

https://wiki.seeedstudio.com/cn/Quantum-Mini-Linux-Development-Kit/

**Comment by hjg66-5** [comment]
hello

**Reply by MatthewJeffson** [reply]
hello
